packages/airbnb-base/package.json

```diff
@@ -24,11 +24,11 @@
   },
   "dependencies": {
     "@neutrinojs/eslint": "^8.2.0",
-    "eslint": "^4.19.1",
     "eslint-config-airbnb-base": "^12.1.0",
     "eslint-plugin-import": "^2.11.0"
   },
   "peerDependencies": {
+    "eslint": "^4.0.0",
```

Comment:
The `@neutrinojs/eslint` package will already have a `peerDependency` on eslint, which will be inherited. Though might be clearer for users inspecting only the top-level package.json to be explicit? I can't decide. (The same also applies to eg the `webpack` peerDep in `@neutrinojs/font-loader` since file-loader has it's own peerDep etc)

Reply:
Yeah, I added this since I figured it would be easier to ensure users would be notified from their top-level peer dependencies, but I am fine with removing some of them if they are bloating.

Reply:
I think given we're now needing users to add a bunch of peerDeps perhaps we need to have it as clear as possible to them :-)